apps/benchmark/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
^ please remove `apps/benchmark/dist/*`

Reply:
Done in [c3375d4](https://github.com/microsoft/griffel/pull/132/commits/c3375d4fc77d4ab50458d45b9de0eb35e1180609)